addins/Cake.AWS.CloudFront.yml

```diff
@@ -4,8 +4,11 @@ Assemblies:
 - "/**/Cake.AWS.CloudFront.dll"
 Repository: https://github.com/SharpeRAD/Cake.AWS.CloudFront
 Author: SharpeRAD
-Description: "AWS CloudFront addin for Cake."
+Description: "Cake Build addon for managing Amazon CloudFront CDN."
 Categories:
-- AWS
-- Deployment
+- Cake

```

Comment:
Common tags like `Cake`, `Build`,  `Script`, `Addin` should be filtered as they server no purpose as categories on the webseite

Reply:
Sounds reasonable, I can filter out these tags

Reply:
Done. These tags will filtered out.